Entwurf/Datenbankdokumentation.tex

```diff
@@ -55,7 +55,7 @@ \section{Datenbank}
     Appoinment & DATE & Der Termin, der durch die Gruppe gebucht oder reserviert ist (Fremdschlüssel auf Appointment.Date). \\ \hline
     \end{tabular}
 \end{table}
-In Tabelle \ref{tab:Reservation} wird die Struktur der Tabelle Reservation  dargestellt.  In dieser Tabelle werden sowohl Reservierungen, als auch Buchungen eingetragen. Eine Buchung ist dadurch zu erkennen, dass es zusätzlich einen entsprechenden Eintrag in Booking gibt. Der Grund für diese Art der Darstellung ist die Erleichterung der Integritätsprüfung, da jeder Termin max. eine Buchung oder Reservierung haben darf, und auch jede Gruppe max. eine Buchung oder Reservierung. Ein Eintrag in Reservation repräsentiert also eine Buchung, wenn zusätzlich ein Eintrag in Booking vorhanden ist, sonst nur eine Reservierung.
+In Tabelle \ref{tab:Reservation} wird die Struktur der Tabelle Reservation  dargestellt.  In dieser Tabelle werden sowohl Reservierungen, als auch Buchungen eingetragen. Eine Buchung ist dadurch zu erkennen, dass es zusätzlich einen entsprechenden Eintrag in Booking gibt. Der Grund für diese Art der Darstellung ist die Erleichterung der Integritätsprüfung, da jeder Termin max. eine Buchung oder Reservierung haben darf, und auch jede Gruppe max. eine Buchung oder Reservierung. Ein Eintrag in Reservation repräsentiert also eine Buchung, wenn zusätzlich ein Eintrag in Booking vorhanden ist, sonst nur eine Reservierung.Darüber hinhaus man kann die Buchung aus Datensicht als Spezialisierung der Reservierung sehen.Jede Buchung verhält sich wie eine Reservierung, aber mit zusätzlichen Eigenschaften.
```

Comment:
Nicht darüber hinaus. Eher 
 > ... Eintrag in Booking gibt. Das liegt daran, dass man eine Buchung aus Datensicht als Spezialisierung der Reservierung sehen kann. Jede Buchung verhält sich wie eine Reservierung, nur mit den zusätzlichen Eigenschaften Startzeit, Endzeit und Raum. Dadurch ist auch die Integritätsprüfung leichter, da jeder Termin max. ...